collect_app/src/main/java/org/odk/collect/android/backgroundwork/autosend/FormLevelAutoSendChecker.kt

```diff
@@ -0,0 +1,17 @@
+package org.odk.collect.android.backgroundwork.autosend
+
+import org.odk.collect.android.utilities.FormsRepositoryProvider
+import org.odk.collect.forms.Form
+
+class FormLevelAutoSendChecker(private val formsRepositoryProvider: FormsRepositoryProvider) {
```

Comment:
I added those two classes to make testing easier otherwise I would end up with plenty tests in `AutoSendTaskSpec` because I would need to verify all those variations where `autosend` is enabled/disabled in settings and enabled/disabled on a form level.